apps/trading/client-pages/market/trade-grid.tsx

```diff
@@ -69,91 +70,85 @@ interface TradeGridProps {
   pinnedAsset?: PinnedAsset;
 }
 
-const MainGrid = ({
-  marketId,
-  onSelect,
-  pinnedAsset,
-}: {
-  marketId: string;
-  onSelect?: (marketId: string) => void;
-  pinnedAsset?: PinnedAsset;
-}) => {
-  const navigate = useNavigate();
-  const onMarketClick = (marketId: string) => {
-    navigate(Links[Routes.MARKET](marketId), {
-      replace: true,
-    });
-  };
-  return (
-    <ResizableGrid vertical>
-      <ResizableGridPanel minSize={75} priority={LayoutPriority.High}>
-        <ResizableGrid proportionalLayout={false} minSize={200}>
-          <ResizableGridPanel
-            priority={LayoutPriority.High}
-            minSize={200}
-            preferredSize="50%"
-          >
-            <TradeGridChild>
-              <Tabs>
-                <Tab id="chart" name={t('Chart')}>
-                  <TradingViews.Candles marketId={marketId} />
-                </Tab>
-                <Tab id="depth" name={t('Depth')}>
-                  <TradingViews.Depth marketId={marketId} />
-                </Tab>
-                <Tab id="liquidity" name={t('Liquidity')}>
-                  <TradingViews.Liquidity marketId={marketId} />
-                </Tab>
-              </Tabs>
-            </TradeGridChild>
-          </ResizableGridPanel>
+const MainGrid = memo(
+  ({
+    marketId,
+    onSelect,
+    pinnedAsset,
+  }: {
+    marketId: string;
+    onSelect?: (marketId: string) => void;
+    pinnedAsset?: PinnedAsset;
+  }) => {
+    const navigate = useNavigate();
+    const { screenSize } = useScreenDimensions();
+    const onMarketClick = useCallback(
+      (marketId: string) => {
+        navigate(Links[Routes.MARKET](marketId), {
+          replace: true,
+        });
+      },
+      [navigate]
+    );
+
+    const bottomPanel = useMemo(() => {
```

Comment:
I can see why we need to memoize here, but I think a more readable pattern would be to make a component here with React.memo. This would make this component more readable

Reply:
done